configuration-as-code-examples/cloudbees-assurance-plugin/casc_disabled.yaml

```
@@ -0,0 +1,10 @@
+# This file shows how it is possible to disable the CloudBees Assurance Program using Configuration as Code.
+# CloudBees does not encourage you to disable it, however.
+beekeeper:
+  enabled: false
+  securityWarnings:
+    enabledForCore: true
```

If core may be confusing, then should we change this?:
```suggestion
    enabledForJenkins: true
```

It may require a new plugin version. If it's not strictly needed, better to keep it as is, @KimberlyFields

---

This is not as easy as changing this @MRamonLeon.

---

This does have the potential to confuse customers since we have a product named Core, but enabledForCore does not refer to CloudBees Core. Can you add some text to explain what it means?